db/migrate/20190227234500_mentor_matchings.rb

```diff
@@ -0,0 +1,11 @@
+class CreateMentorMatchings < ActiveRecord::Migration
+    def change
+      create_table :mentor_matchings do |t|
+        t.integer :team_id
+        t.integer :mentor_id
+        t.integer :choice_ranking
+        t.boolean :mentor_accepted
+        t.integer :cohort
+      end
+    end
+end 
```

Comment:
From the model it seems like it belongs to `:team` and `:mentor`, are you missing the references to these model?